flows/moped/prefect_agent_environment_variables.env_template

```diff
@@ -30,3 +30,9 @@ ELB_SECURITY_GROUP=
 ECS_TASK_SECURITY_GROUP=
 TASK_ROLE_ARN=
 CLOUDWATCH_LOG_GROUP=
+
+NETLIFY_BUILD_HOOK=
+NETLIFY_ACCESS_TOKEN=
+NETLIFY_SITE_ID=
+NETLIFY_API_URL=
+NETLIFY_SEARCH_BACK_DISTANCE_FIND_BUILDS=1000
```

Comment:
This value having a suggested value of 1000 is reasonable. It's the number of netlify deployments in which you would expect a branch's previous deployment to be found, if there is one.